aries_cloudagent/config/argparse.py

```diff
@@ -683,6 +683,13 @@ class TransportGroup(ArgumentGroup):
 
     def add_arguments(self, parser: ArgumentParser):
         """Add transport-specific command line arguments to the parser."""
+        parser.add_argument(
+            "--mediation",
+            metavar="<label>",
+            help="Enables didcomm mediation. After establishing a connection, if enabled, \
+                an agent may request message mediation, which will allow the mediator to \
+                forward messages on behalf of the recipient. See aries-rfc:0211.",
+        )
```

Comment:
Am I reading this incorrectly or do you have two different arguments being added with basically the same values? These lines and 767-793

Reply:
yup, that it is. let us never talk about this again.

Reply:
I see, I may have misread. I'm not sure if the argument found here makes sense or not. Activating mediation is basically equivalent to enabling the coordinate-mediation protocol so I'm inclined to say that we don't need the separate `--mediation` flag to turn on mediation. For now, I think we should keep it to just the `--open-mediation` flag and we can expand from there in the future as needed.

Reply:
I did not know what group to place the flag. I originally placed a --mediation flag in transport(which I just finished removing). At which point mediation did not feel like a "transport" so I created a new group for mediation. I liked this especially because we could have more configurations for queueing messages and terms of mediation. This may be premature optimization. 
(나는 우리가 이것을 다시는 말하지 않을 것이라는 인상을 받았다.)

Reply:
I think the group is likely appropriate. Just the separate `--mediation` flag was where I took issue. Thanks for the fix.

(ㅋㅋㅋ)